### PR TITLE
Update what-is-microsoft-defender-threat-intelligence-defender-ti.md

### PR DESCRIPTION
**Pull request by gichiu**
Replaced the word artifact with indicator.  Made updates to 2 paragraphs.

### CODE REVIEW

**Comment by gichiu** [comment]
@Dansimp, this is my first time editing.  Do I need to tag someone to review and approve?

File: defender/threat-intelligence/what-is-microsoft-defender-threat-intelligence-defender-ti.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ In the same breadth, Vulnerability Intelligence Analysts battle correlating thei
...
 
-Below is a screenshot of Defender TI’s Threat Intelligence Home Page. Analysts can quickly scan new featured articles as well as begin their intelligence gathering, triage, incident response, and hunting efforts by performing a keyword, artifact or CVE-ID search.
+Below is a screenshot of Defender TI’s Threat Intelligence Home Page. Analysts can quickly scan new featured articles as well as begin their intelligence gathering, triage, incident response, and hunting efforts by performing a keyword, indicator or CVE-ID search.
```
Comment (JohanFreelancer9) [inline defender/threat-intelligence/what-is-microsoft-defender-threat-intelligence-defender-ti.md]:
```suggestion
Below is a screenshot of Defender TI’s Threat Intelligence Home Page. Analysts can quickly scan new featured articles as well as begin their intelligence gathering, triage, incident response, and hunting efforts by performing a keyword, indicator, or CVE-ID search.
```

File: defender/threat-intelligence/what-is-microsoft-defender-threat-intelligence-defender-ti.md [hunk 2]
```diff
@@ -121,9 +121,9 @@ For more information, see [Using tags](using-tags.md).
 
 ## Projects
 
-Microsoft’s Defender TI platform allows users to develop multiple project types for organizing indicators of interest and indicators of compromise from an investigation. Projects contain a listing of all associated artifacts and a detailed history that retains the names, descriptions, and collaborators.
+Microsoft’s Defender TI platform allows users to develop multiple project types for organizing indicators of interest and indicators of compromise from an investigation. Projects contain a listing of all associated indicators and a detailed history that retains the names, descriptions, and collaborators.
```
Comment (JohanFreelancer9) [inline defender/threat-intelligence/what-is-microsoft-defender-threat-intelligence-defender-ti.md]:
```suggestion
Microsoft Defender TI platform allows users to develop multiple project types for organizing indicators of interest and indicators of compromise from an investigation. Projects contain a listing of all associated indicators and a detailed history that retains the names, descriptions, and collaborators.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @gichiu, kindly commit the two minor additional suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@DulceMontemayor Please merge this PR. Thanks

**Pull request merged**